### PR TITLE
Remove TaskDialog code

### PR DESCRIPTION
**Pull request by paulvanbrenk**
We were unable to make them accessible, and they didn't add much value.
* For the Project Wizard, the assemlby is installed by default
* The NodeModules should never be added to the project
* The Max Path problem is solved with NPM 4+

### CODE REVIEW

File: Nodejs/Product/Nodejs/Project/NodejsFolderNode.cs [hunk 1]
```diff
@@ -29,37 +33,18 @@ internal override int IncludeInProject(bool includeChildren)
...
+            if (CommonUtils.IsSameDirectory(nodeModulesPath, this.ItemNode.Url))
             {
+                ShouldIncludeNodeModulesFolderInProject();
                 return VSConstants.S_OK;
             }
             return base.IncludeInProject(includeChildren);
         }
 
-        private bool ShouldIncludeNodeModulesFolderInProject()
+        private void ShouldIncludeNodeModulesFolderInProject()
```
Comment (amcasey) [inline Nodejs/Product/Nodejs/Project/NodejsFolderNode.cs]: This name no longer makes sense.

File: Nodejs/Product/Nodejs/Project/NodejsProjectNode.cs [hunk 2]
```diff
@@ -586,45 +583,9 @@ internal struct LongPathInfo
...
-                            Process.Start("https://go.microsoft.com/fwlink/?LinkId=456511");
-                            break;
-                        default:
-                            System.Windows.Clipboard.SetText(e.Url);
-                            break;
-                    }
-                };
 
-                recheck:
+                Utilities.ShowMessageBox(
```
Comment (amcasey) [inline Nodejs/Product/Nodejs/Project/NodejsProjectNode.cs]: I might be missing something, but it looks like we show the warning *before* doing the long path check.

Comment (amcasey) [inline Nodejs/Product/Nodejs/Project/NodejsProjectNode.cs]: Also, this doesn't obviously contain the path causing the problem.

File: Nodejs/Product/Nodejs/SharedProject/ProjectNode.CopyPaste.cs [hunk 3]
```diff
@@ -1113,7 +1113,6 @@ private Addition CanAddFileFromProjectReference(string projectRef, string target
                         {
                             throw;
                         }
-                        TaskDialog.ForException(this.Project.Site, e, string.Empty, this.Project.IssueTrackerUrl).ShowModal();
```
Comment (amcasey) [inline Nodejs/Product/Nodejs/SharedProject/ProjectNode.CopyPaste.cs]: No message box for this one?

Comment (paulvanbrenk) [reply Nodejs/Product/Nodejs/SharedProject/ProjectNode.CopyPaste.cs]: This only pointed the user to the a new issue page... which, without a stack or a repo project is pointless.

**Pull request merged**